growth/too/templates/plan_new.html

```diff
@@ -65,6 +65,10 @@
                         {{ render_field(form.primary, class='custom-control-input') }}
                         <label for=primary class=custom-control-label>Use primary only</label>
                     </div>
+                    <div class="custom-control custom-switch">
+                        {{ render_field(form.balance, class='custom-control-input') }}
+                        <label for=balance class=custom-control-label>Filter balancing</label>
```

Comment:
I'm not 100% clear on what filter balancing means. It would be helpful to add a tool tip or form text to explain what it does.

Reply:
I was more verbose!